Issue #74 Majority Functionality of Journal Calendar with Dummy Data

maheknangia opened the pull request:
Issue #74

## Proposed Changes

- Created a day calendar with the ability to add and edit events
- Dummy data was used to see if events planned in advance show up on calendar
- Delete event still needs to be implemented 

Screenshots:
![Screen Shot 2021-03-21 at 1 13 18 PM](https://user-images.githubusercontent.com/63040474/111889117-61059900-8a47-11eb-85fc-32b73e04b30f.png)
![Screen Shot 2021-03-21 at 2 10 15 PM](https://user-images.githubusercontent.com/63040474/111890546-391a3380-8a4f-11eb-8bc1-7336c4fa5975.png)

maheknangia committed:
changed index.js to run App

maheknangia committed:
different colours for priorities show up for full month

maheknangia committed:
different colours for priorities show up for full month

maheknangia committed:
different colours for priorities show up for full month

maheknangia committed:
different colours for priorities show up for full month

maheknangia committed:
shows as day view, required colours set, delete & edit now saving

maheknangia committed:
shows as day view, required colours set, delete & edit now saving

maheknangia committed:
shows as day view, required colours set, delete & edit now saving

maheknangia committed:
shows as day view, required colours set, delete & edit now saving

maheknangia committed:
event names not showing up

maheknangia committed:
event names not showing up

maheknangia committed:
event names not showing up

maheknangia committed:
event names not showing up

maheknangia committed:
changed colours for locations

maheknangia committed:
changed colours for locations

maheknangia committed:
changed colours for locations

maheknangia committed:
changed colours for locations

maheknangia committed:
names of events show up as data

maheknangia committed:
names of events show up as data

maheknangia committed:
names of events show up as data

maheknangia committed:
names of events show up as data

maheknangia committed:
events can be edited

maheknangia committed:
events can be edited

maheknangia committed:
events can be edited

maheknangia committed:
events can be edited

maheknangia committed:
delete isn't deleting

maheknangia committed:
delete isn't deleting

maheknangia committed:
delete isn't deleting

maheknangia committed:
delete isn't deleting

maheknangia committed:
removed drag and drop

maheknangia committed:
removed drag and drop

maheknangia committed:
removed drag and drop

maheknangia committed:
removed drag and drop

maheknangia committed:
refactored

maheknangia committed:
refactored

maheknangia committed:
refactored

maheknangia committed:
refactored

maheknangia committed:
ready for demo

maheknangia committed:
ready for demo

maheknangia committed:
ready for demo

maheknangia committed:
ready for demo

maheknangia committed:
refactored file

maheknangia committed:
refactored file

maheknangia committed:
refactored file

maheknangia committed:
refactored file

maheknangia committed:
changed current date

maheknangia committed:
changed current date

maheknangia committed:
changed current date

maheknangia committed:
changed current date

maheknangia committed:
Merge remote-tracking branch 'origin/main' into main

maheknangia committed:
Merge branch 'Functionality' into main

maheknangia committed:
package files updated

maheknangia committed:
package files updated

maheknangia committed:
package files updated

maheknangia committed:
package files updated

RawiriHohepa committed:
Add Calendar and Text journal widgets to Journal (#178)

RawiriHohepa commented on telos-frontend/src/components/calendar/JournalCalendar.jsx:
```diff
@@ -1,162 +1,153 @@
+    color: '#0000de',
+  },
+];
+export default class Calendar extends React.PureComponent {
```
What's your reasoning for using a class here instead of a functional component (`export default function Calendar()`)?

maheknangia replied:
I had been quite stuck on being able to implement the colour changes due to priority in the functional component so I did a google search and they recommended I try using a class instead.

cng822 committed:
Closes #51: To Do List (Journal / Daily View) - Functionality with dummy data (#169)

* implemented design todo list

* Update index.js

* Update index.js

* Update index.js

* Update index.js

* added delete/cancel features

* added dummy data

* change syntax

* deleted function updated

* fixed delete

* updated for modal

* added modals for migrate

* minor changes

* additional style changes

* Changed the Journal Calendar colour from blue to purple (#154)

* Implement a theme and Theme Provider

Created a theme which changes the colour from blue to purple and a provider to implement the changes to on the app.

* Formatting Fix

Removed spaces before  and after the '=' in the theme provider

* Fix components reaching past screen size then requiring scrolling (#164)

* Closes #55: Habit tracker (Journal / Daily View) - Functionality w/ dummy data (#153)

* typo fixed and hover effect

* add vs code ignore

* rm root folder package-lock.json

* Partial function

* implement functions

* Update addbutton style

* reuse sub-component and merge imports

* Fixed static repeated port for express app in integration tests (#168)

* added a text input for due date in migrate

* implemented design todo list

* Update index.js

* Update index.js

* added delete/cancel features

* added dummy data

* change syntax

* deleted function updated

* fixed delete

* updated for modal

* added modals for migrate

* minor changes

* additional style changes

* added a text input for due date in migrate

* changed css file to module.css

* fixed component styling issue

Co-authored-by: detz582 <48495086+ODE-BOY@users.noreply.github.com>
Co-authored-by: Rawiri Hohepa <54158903+RawiriHohepa@users.noreply.github.com>
Co-authored-by: ysway <syan553@aucklanduni.ac.nz>
Co-authored-by: Felix Yang <49087744+flexzy@users.noreply.github.com>

ambiyu committed:
Implemented page specific widgets (#181)

maheknangia committed:
Merge remote-tracking branch 'origin/main' into main

# Conflicts:
#	telos-frontend/src/components/calendar/JournalCalendar.jsx

maheknangia committed:
Merge remote-tracking branch 'origin/main' into main

# Conflicts:
#	telos-frontend/package-lock.json
